packages/app-store/office365calendar/lib/CalendarService.ts

```diff
@@ -55,11 +55,11 @@ export default class Office365CalendarService implements Calendar {
 
   async createEvent(event: CalendarEvent): Promise<NewCalendarEventType> {
     try {
-      const calendarId = event.destinationCalendar?.externalId
-        ? `${event.destinationCalendar.externalId}/`
-        : "";
+      const eventsUrl = event.destinationCalendar?.externalId
```

Comment:
See [docs](https://learn.microsoft.com/en-us/graph/api/calendar-post-events?view=graph-rest-1.0&tabs=http).

We were generating a URL of `/me/calendars/events` when there's no externalId but the URL needs to be `/me/calendar/events` in that case. Upon debugging, I saw that Outlook 365 was sending us back a 405.

<img width="790" alt="Screen Shot 2023-03-14 at 10 53 03 AM" src="https://user-images.githubusercontent.com/2538462/225040330-cfbfc5ce-a644-427e-83a0-705e2ed10bcf.png">

Reply:
I'm surprised this is showing up now. How did you manage to get the 405 response?

Reply:
I manually added a console log to see the response. I'll probably spend some more time to see why if these errors aren't being logged currently.